app/forms/pages/delete_condition_form.rb

```diff
@@ -0,0 +1,24 @@
+class Pages::DeleteConditionForm
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :form, :page, :check_page_id, :routing_page_id, :answer_value, :goto_page_id, :record, :confirm_deletion
+
+  validates :confirm_deletion, presence: true, inclusion: { in: %w[true false] }
+
+  def delete
+    return false if invalid?
+
+    result = true
+
+    if confirm_deletion == "true"
+      result = record.destroy
+    end
+
+    result
+  end
+
+  def goto_page_options
+    [OpenStruct.new(id: nil, question_text: I18n.t("helpers.label.pages_conditions_form.default_goto_page_id")), form.pages.map { |p| OpenStruct.new(id: p.id, question_text: p.question_text) }].flatten
+  end
```

Comment:
This code looks to be the same as what is in the `Pages::ConditionsForm` class... can we somehow include that method in this class, to avoid the repetition?

Reply:
I've changed it so that it's a better suited method for the delete form - please could you review @lfdebrux

Reply:
I've reviewed this again and it looks good to merge